assets/js/state/clusterChecksResultsFilters.js

```diff
@@ -0,0 +1,17 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {};
+
+export const clusterChecksResultsFiltersSlice = createSlice({
+  name: 'clusterChecksResultsFilters',
+  initialState,
+  reducers: {
+    setClusterFilters: (state, { payload: { clusterID, filters } }) => {
```

Comment:
What about making this generic `setChecksResultsFilters`.
Getting the `{ id: filters }` payload. I'm just thinking for future uses where we don't only have checks for clusters.
All of our resouces (clusters, hosts, sap systesms) are identified by uuid, so there wouldn't be any collision.

Reply:
Right again :D